src/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -0,0 +1,39 @@
+import {
+  Breadcrumbs as MuiBreadcrumbs,
+  breadcrumbsClasses
+} from "@mui/material";
+
+import { BreadcrumbsProps } from "./Breadcrumbs.types";
+import NavigateNextIcon from "@mui/icons-material/NavigateNext";
+import React from "react";
+
+/**
+ * Breadcrumbs component
+ */
+function Breadcrumbs({ children, ...rest }: BreadcrumbsProps) {
+  return (
+    <MuiBreadcrumbs
+      {...rest}
+      aria-label="breadcrumbs"
+      separator={<NavigateNextIcon fontSize="small" />}
+      // style overrides to style children
+      sx={{
+        [`& .${breadcrumbsClasses.li} *`]: {
+          color: "text.secondary",
+          textDecoration: "none"
+        },
+        [`& .${breadcrumbsClasses.li}:last-child *`]: {
+          color: "text.primary",
+          textDecoration: "none"
+        },
+        [`& .${breadcrumbsClasses.li}:not(:last-child):hover *`]: {
+          textDecoration: "underline"
+        }
```

Comment:
There is some specific styling in [TEST](https://github.com/IPG-Automotive-UK/virto/blob/805c56b66408e4a11037527a67c48110fd831135/apps/test/app/routes/tests.%24id/route.tsx#L103-L116) to ensure that breadcrumbs can never flow over multiple lines. I think this is useful for all apps to protect against long document names:

TEST:
![image](https://github.com/IPG-Automotive-UK/react-ui/assets/27866636/1902d680-23f5-4c17-8260-8654c62095a9)
Storybook:
![image](https://github.com/IPG-Automotive-UK/react-ui/assets/27866636/d4521272-0930-4a2f-9127-b143aec0b728)

Reply:
I've updated this but changed it slightly, I found the example in scene. I noticed that you can still break the breadcrumbs as it was only constraining the last child. I've updated this to allow the constraint of every breadcrumb.

Reply:
Nice, I don't think we currently have a VIRTO case where we have more than two breadcrumbs but nice attention to detail 👊